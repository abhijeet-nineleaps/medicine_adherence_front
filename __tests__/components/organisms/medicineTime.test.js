import Enzyme, {shallow} from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import React from 'react';
import Box from '../../../src/components/organisms/medicineTime';

Enzyme.configure({adapter: new Adapter()});

describe('test collector category', () => {
  it('test category', () => {
    let time = [{}, {}];
    const wrapper = shallow(<Box {...time} />)
      .childAt(0)
      .dive();
    wrapper.find('#box').props().onPress();
  });
});
