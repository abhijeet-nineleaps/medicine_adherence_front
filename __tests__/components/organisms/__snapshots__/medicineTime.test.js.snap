// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test collector category test category 1`] = `
<TouchableOpacity
  disableBuiltInState={true}
  disabled={false}
  fillColor="#3743ab"
  iconStyle={
    Object {
      "borderColor": "#3743ab",
      "borderWidth": 1.3,
    }
  }
  id="box"
  isChecked={false}
  onPress={[Function]}
  size={22}
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      },
      undefined,
    ]
  }
  textStyle={
    Object {
      "color": "black",
      "fontFamily": "JosefinSans-Regular",
      "fontSize": 17,
    }
  }
  unfillColor="#FFFFFF"
>
  <ForwardRef(AnimatedComponentWrapper)
    style={
      Array [
        Object {
          "transform": Array [
            Object {
              "scale": 1,
            },
          ],
        },
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "borderColor": "#3743ab",
          "borderRadius": 11,
          "borderWidth": 1,
          "height": 22,
          "justifyContent": "center",
          "width": 22,
        },
        Object {
          "borderColor": "#3743ab",
          "borderWidth": 1.3,
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "marginLeft": 16,
        },
        undefined,
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#757575",
            "fontSize": 16,
            "textDecorationLine": "none",
          },
          Object {
            "color": "black",
            "fontFamily": "JosefinSans-Regular",
            "fontSize": 17,
          },
        ]
      }
    />
  </View>
</TouchableOpacity>
`;
